src/ui/registration/components/seedPhrase.tsx

```diff
@@ -14,88 +13,66 @@ interface Props {
 }
 
 const styles = StyleSheet.create({
-  title: {
-    color: JolocomTheme.primaryColorSand,
-    fontSize: JolocomTheme.labelFontSize
+  container: {
+    backgroundColor: JolocomTheme.primaryColorBlack
   },
   phrase: {
-    color: JolocomTheme.primaryColorSand,
-    fontSize: JolocomTheme.labelFontSize
+    paddingLeft: '3%',
+    paddingRight: '3%',
+    color: JolocomTheme.primaryColorWhite,
+    fontSize: 34,
+    fontFamily: JolocomTheme.contentFontFamily,
+    lineHeight: 38
   },
-  sideNoteGreen: {
+  sideNote: {
+    paddingLeft: '5%',
+    paddingRight: '5%',
+    lineHeight: 26,
     color: JolocomTheme.primaryColorSand,
-    fontSize: JolocomTheme.labelFontSize
-  }
-})
-
-const muiStyles = {
-  button: {
-    container: {
-      backgroundColor: JolocomTheme.primaryColorBlack
-    },
+    fontSize: JolocomTheme.labelFontSize,
+    fontFamily: JolocomTheme.contentFontFamily
+  },
+  footerButton: {
+    position: 'absolute',
+    bottom: '5%'
   },
-  checkbox: {
-    label: {
-      marginLeft: -5,
-      padding: 0,
-      position: 'relative',
-      textAlign: 'left',
-      color: JolocomTheme.primaryColorPurple,
-      fontSize: JolocomTheme.labelFontSize,
-    },
-    icon: {
-      color: JolocomTheme.primaryColorSand
-    }
+  buttonContainer: {
```

Comment:
Ideally we can use flex / `height: 100%` instead of explicit values.
We can perhaps have an issue / pr that takes care of this in all components.